packages/PHPStanStaticTypeMapper/TypeMapper/BooleanTypeMapper.php

```diff
@@ -15,6 +15,9 @@
 use Rector\Core\ValueObject\PhpVersionFeature;
 use Rector\PHPStanStaticTypeMapper\Contract\TypeMapperInterface;
 
+/**
+ * @implements TypeMapperInterface<BooleanType>
+ */
 final class BooleanTypeMapper implements TypeMapperInterface
```

Comment:
:+1:

Reply:
Any plan to update this? The `@implement` part and changes in `TypeMapperInterface` are ready to merge. The rest must be reverted to PhpStorm understands the code

Reply:
Will do tomorrow

Reply:
Thanks!

Reply:
@TomasVotruba done